README.md

```diff
@@ -99,6 +98,14 @@ from your root directory to give the bash file executable permission.
 ```
 This also runs the tailwind watcher
 
+### Convert the source XLSX data source to JSON
+
+Taking an input XLSX file similar to the one in this repository, you can convert it to
+JSON by running
+```shell
+python -m app.xlsx2json
```

Comment:
@MRigal Does this work without the `.py` suffix at the end of the file name?

Reply:
Yes, it's how `-m` works. Only without .py suffix